catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/geofeature/GazetteerFeatureServiceTest.java

```diff
@@ -60,9 +62,14 @@
           .alternateNames("")
           .build();
 
-  private static final List<GeoEntry> QUERYABLE_RESULTS = Arrays.asList(GEO_ENTRY_1, GEO_ENTRY_2);
+  private static final List<Suggestion> SUGGESTED_NAMES = new ArrayList<>();
 
-  private static final List<String> SUGGESTED_NAMES = Arrays.asList("name1", "name2");
+  static {
```

Comment:
:pencil2: Could put this in a `@BeforeClass` method if you didn't want to use a static block.